javaxval/intelliJ-formatter.xml

```diff
@@ -0,0 +1,39 @@
+<code_scheme name="baeldung-formatter" version="173">
+    <option name="RIGHT_MARGIN" value="260"/>
+    <option name="FORMATTER_TAGS_ENABLED" value="true"/>
+    <JavaCodeStyleSettings>
+        <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="500"/>
+        <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="300"/>
+        <option name="ENABLE_JAVADOC_FORMATTING" value="false"/>
+    </JavaCodeStyleSettings>
+    <codeStyleSettings language="JAVA">
+        <option name="KEEP_LINE_BREAKS" value="false"/>
+        <option name="KEEP_FIRST_COLUMN_COMMENT" value="false"/>
+        <option name="KEEP_CONTROL_STATEMENT_IN_ONE_LINE" value="false"/>
+        <option name="KEEP_BLANK_LINES_IN_DECLARATIONS" value="1"/>
+        <option name="KEEP_BLANK_LINES_IN_CODE" value="1"/>
+        <option name="KEEP_BLANK_LINES_BEFORE_RBRACE" value="1"/>
+        <option name="INDENT_CASE_FROM_SWITCH" value="false"/>
+        <option name="ALIGN_MULTILINE_PARAMETERS" value="false"/>
+        <option name="ALIGN_MULTILINE_RESOURCES" value="false"/>
+        <option name="SPACE_WITHIN_ARRAY_INITIALIZER_BRACES" value="true"/>
+        <option name="SPACE_BEFORE_ARRAY_INITIALIZER_LBRACE" value="true"/>
+        <option name="CALL_PARAMETERS_WRAP" value="1"/>
+        <option name="METHOD_PARAMETERS_WRAP" value="1"/>
+        <option name="RESOURCE_LIST_WRAP" value="5"/>
+        <option name="EXTENDS_LIST_WRAP" value="1"/>
+        <option name="THROWS_LIST_WRAP" value="1"/>
+        <option name="EXTENDS_KEYWORD_WRAP" value="1"/>
+        <option name="THROWS_KEYWORD_WRAP" value="1"/>
+        <option name="METHOD_CALL_CHAIN_WRAP" value="5"/>
+        <option name="BINARY_OPERATION_WRAP" value="1"/>
+        <option name="BINARY_OPERATION_SIGN_ON_NEXT_LINE" value="true"/>
+        <option name="TERNARY_OPERATION_WRAP" value="5"/>
+        <option name="ARRAY_INITIALIZER_WRAP" value="1"/>
+        <option name="VARIABLE_ANNOTATION_WRAP" value="2"/>
+        <indentOptions>
+            <option name="CONTINUATION_INDENT_SIZE" value="2"/>
+            <option name="SMART_TABS" value="true"/>
+        </indentOptions>
+    </codeStyleSettings>
+</code_scheme>
```

Comment:
Do we need this file adding to this module?

Reply:
No, I'll remove it.